linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -114,6 +114,36 @@ trait HttpClientConfig extends ClientConfig {
   }
 }
 
+@JsonTypeInfo(
+  use = JsonTypeInfo.Id.NAME,
+  include = JsonTypeInfo.As.EXISTING_PROPERTY,
+  property = "kind",
+  visible = true,
+  defaultImpl = classOf[HttpDefaultSvc]
+)
+@JsonSubTypes(Array(
+  new JsonSubTypes.Type(value = classOf[HttpDefaultSvc], name = "io.l5d.global"),
+  new JsonSubTypes.Type(value = classOf[HttpStaticSvc], name = "io.l5d.static")
+))
+abstract class HttpSvc extends Svc
+
+class HttpDefaultSvc extends HttpSvc with DefaultSvc with HttpSvcConfig
```

Comment:
i think this makes it clear that DefaultSvc should be `trait DefaultSvc { _: Svc => ... }`